src/UserSpaceInstrumentation/InjectLibraryInTracee.cpp

```diff
@@ -81,9 +81,18 @@ ErrorMessageOr<uint64_t> ExecuteOrDie(pid_t pid, uint64_t address_code, uint64_t
   RegisterState original_registers;
   OUTCOME_TRY(original_registers.BackupRegisters(pid));
 
-  RegisterState registers_set_rip = original_registers;
-  registers_set_rip.GetGeneralPurposeRegisters()->x86_64.rip = address_code;
-  OUTCOME_TRY(registers_set_rip.RestoreRegisters());
+  RegisterState registers_for_execution = original_registers;
+  registers_for_execution.GetGeneralPurposeRegisters()->x86_64.rip = address_code;
+  // The calling convention for x64 assumes the 128 bytes below rsp to be usable as a scratch pad
```

Comment:
nit. "128 bytes below rsp" might be confusing given that stack addresses decrease as we push. Consider changing to "128 bytes below rsp" or maybe use something like "The calling convention for x64 assumes a scratch pad of 128 bytes for the current function.

Reply:
I guess it depends on when you look at rsp, I'd say that if it's from the perspective of the called function, then "above" should be used. From the caller, "below" is fine. Feel free to disregard.

Reply:
It is "below" from the perspective of the called function. Unlike what is done on Windows the function might access rsp-8, rsp-16, ... without changing rsp. Typically only leaf functions will do that.

Reply:
What about using "after" instead? It be slightly clearer that it means in the direction in which the stack grows.

Reply:
I'll keep the "below". "after" sounds a bit like "toward larger numbers" to me.